index.js

```diff
@@ -0,0 +1,35 @@
+const hazyCalculator = (calculationSteps) => {
+  let operation
+
+  // Find Operation
+  for (const item of calculationSteps) {
+    if (item === '*' || item === '+' || item === '-' || item === '/') {
```

Comment:
assuming array only has one operator. this will work fine. and also it will always pick the last operator provided.

we will assume its only one.